AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/ModelSyncedEventEmitter.swift

```diff
@@ -134,8 +134,13 @@ final class ModelSyncedEventEmitter {
                                                  data: modelSyncedEventBuilder.build())
         Amplify.Hub.dispatch(to: .dataStore, payload: modelSyncedEventPayload)
         modelSyncedEventTopic.send(completion: .finished)
+        cancel()
     }
 
+    func cancel() {
+        syncOrchestratorSink?.cancel()
+        reconciliationQueueSink?.cancel()
+    }
```

Comment:
These two should be cancelled manually, because they never receives completion from downstream from my finding